packages/autorest.python/autorest/codegen/models/enum_type.py

```diff
@@ -45,6 +46,87 @@ def from_yaml(
         )
 
 
+class EnumValueType(BaseType):
+    def __init__(
+        self,
+        yaml_data: Dict[str, Any],
+        code_model: "CodeModel",
+        value_type: BaseType,
+    ) -> None:
+        super().__init__(yaml_data=yaml_data, code_model=code_model)
+        self.name: str = yaml_data["name"]
+        self.value: str = self.yaml_data["value"]
+        self.enum_name: str = yaml_data["enumName"]
```

Comment:
also we already have `EnumValue`, we don't need a new type. I can take some time and do a PR today since I've already created a PR to expose the parent enum on the `SdkEnumValueType` https://github.com/Azure/typespec-azure/pull/3723